binding-library/java/pom.xml

```diff
@@ -0,0 +1,140 @@
+		<version>8.0.1</version>
+	</parent>
+
+    <name>Microsoft Azure Functions Java RabbitMQ types</name>
+    <description>This package contains all Java interfaces and annotations to interact with Microsoft Azure Functions runtime for RabbitMQ Service.</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven-compiler.version>3.8.0</maven-compiler.version>
+        <maven-source.version>3.0.1</maven-source.version>
+        <maven-javadoc.version>3.0.1</maven-javadoc.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <azure.functions.java.library.version>1.3.0</azure.functions.java.library.version>
+    </properties>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:https://github.com/Azure/azure-functions-rabbitmq-extension</connection>
+        <developerConnection>scm:git:git@github.com:Azure/azure-functions-rabbitmq-extension</developerConnection>
+        <url>https://github.com/Azure/azure-functions-rabbitmq-extension</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <distributionManagement>
+      <snapshotRepository>
+        <id>ossrh</id>
+        <name>Sonatype Snapshots</name>
+        <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        <uniqueVersion>true</uniqueVersion>
+        <layout>default</layout>
+      </snapshotRepository>
+    </distributionManagement>
+
+    <developers>
+        <developer>
+            <id>pragnagopa</id>
+            <name>Pragna Gopa</name>
+            <email>pgopa@microsoft.com</email>
+        </developer>
+        <developer>
+            <id>cachai2</id>
+            <name>Cary Chai</name>
+            <email>cachai@microsoft.com</email>
+        </developer>
+        <developer>
+            <id>yojagad</id>
+            <name>Yogesh Jagadeesan</name>
+            <email>yojagad@microsoft.com</email>
+        </developer>
+    </developers>
+
+    <repositories>
+        <repository>
+            <id>maven.snapshots</id>
+            <name>Maven Central Snapshot Repository</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure.functions</groupId>
+            <artifactId>azure-functions-java-library</artifactId>
+            <version>${azure.functions.java.library.version}</version>
+        </dependency>
+        <dependency>
+			<groupId>junit</groupId>
```

Comment:
fomatting still looks off?

Reply:
yeah, looks like I was still using tabs in some places resulting in the format